src/Lentille.hs

```diff
@@ -203,3 +206,6 @@ isChangeTooOld date (change, _) =
   case changeUpdatedAt change of
     Just changeDate -> T.toUTCTime changeDate < date
     _ -> True
+
+swapDuration :: ChangeOptionalDuration -> ChangeEventOptionalDuration
```

Comment:
Perhaps a `From ChangeOptionalDuration ChangeEventOptionalDuration` instance would be more appropriate?

Reply:
Both type are from codegen, and a From instance will raise a orphan instance warning.